packages/browser-sdk/src/stories/prebuilt-ui.stories.tsx

```diff
@@ -79,24 +79,33 @@ const WherebyEmbed = ({
     const elmRef = useRef<WherebyEmbedElement>(null);
     const [cameraEnabled, setCameraEnabled] = useState(video);
     const [meetingEnded, setMeetingEnded] = useState(false);
+    const [grantedDevicePermissions, setGrantedDevicePermissions] = useState(false);
 
     useEffect(() => {
         const element = elmRef.current;
 
         element?.addEventListener("camera_toggle", (e) => {
             const cameraEnabled = e.detail.enabled;
+            console.log(cameraEnabled);
```

Comment:
```suggestion
```

Reply:
haha nice catch. stuff we put in to test with and I just forgot to remove!